docs/user/alerting/stack-alerts/index-threshold.asciidoc

```diff
@@ -48,71 +48,56 @@ When the index threshold alert condition is met, the following variables are ava
 `context.value`:: The value for the alert that met the threshold condition.
 `context.conditions`:: A description of the threshold condition. Example: `count greater than 4`
 
+<<defining-alerts-actions-details, Add an action>> to the alert to use these variables.
+
 [float]
 ==== Example
 
 In this section, you will use the {kib} <<add-sample-data, weblog sample dataset>> to setup and tune the conditions on an index threshold alert. For this example, we want to detect when any of our top three sites have served more than 420,000 bytes over a 24 hour period.
 
-From the <<alert-management, alert management UI>>, create a new alert, and fill in the <<defining-alerts-general-details, general alert details>>. This alert will be checked every 4 hours, and will not execute actions more than once per day. Choose the index threshold alert type. 
-
+. From the <<alert-management, Alerts and Actions>>, create a new alert and fill in the <<defining-alerts-general-details, general alert details>>. This alert will be checked every 4 hours, and will only execute actions when the alert status changes. Choose the index threshold alert type. 
```

Comment:
How about this for the first step.

1. Open the main menu, then click **Stack Management > Alerts and Actions**
2. Create a new alert that is checked every four hours and executes when the alert status changes.
3. Select the **Index threshold** alert type.
```suggestion
.  Open the main menu, then click **Stack Management > Alerts and Actions**.
.  Create a new alert that is checked every four hours and executes when the alert status changes.
.  Select the **Index threshold** alert type.
```

Reply:
If you use this, please verify that the formatting is correct.